doc/code_examples/Tutorial_GUI_ParamEditor.cpp

```diff
@@ -58,6 +58,6 @@ Int main(int argc, const char** argv)
 
   editor->store();
   paramFile.store("Tutorial_ParamEditor_out.ini", param);
-
+  delete editor;
```

Comment:
nice!

But: 
what happens if an exception is thrown between lines 54 and 60?

-->
better solution (since there is no need for the heap allocation in the first place):
```
  ParamEditor editor(nullptr);
  editor.load(param);
  editor.show();
  app.exec();

  editor.store();
  paramFile.store("Tutorial_ParamEditor_out.ini", param);
```